### PR TITLE
chore(demos): fix sidenav styles

### PR DESCRIPTION
**Pull request by kara**
r: @jelbourn 

Small PR to fix the sidenav styles in the demo.

### CODE REVIEW

File: src/demo-app/demo-app.scss [hunk 1]
```diff
@@ -36,3 +35,18 @@
 .demo-content {
   padding: 15px;
 }
+
+md-sidenav-layout {
```
Comment (jelbourn) [inline src/demo-app/demo-app.scss]: Prefix with `demo-`

Comment (kara) [inline src/demo-app/demo-app.scss]: Always forget!  Will do

File: src/demo-app/demo-app.scss [hunk 2]
```diff
@@ -31,8 +29,25 @@
     font-size: 20px;
   }
 
-}
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
```
Comment (jelbourn) [inline src/demo-app/demo-app.scss]: All the styles that apply directly `demo-root` should go right under the definition, before the nested selectors.

Comment (kara) [inline src/demo-app/demo-app.scss]: Done.

**Comment by jelbourn** [comment]
LGTM

**Pull request merged**